resources/definitions/fdmprinter.def.json

```diff
@@ -699,6 +699,20 @@
                                 }
                             }
                         },
+                        "topmost_skin_line_width":
+                        {
+                            "label": "Roofing Line Width",
+                            "description": "Width of a single line of the areas at the top of the print.",
+                            "unit": "mm",
+                            "minimum_value": "0.001",
+                            "minimum_value_warning": "0.1 + 0.4 * machine_nozzle_size",
+                            "maximum_value_warning": "2 * machine_nozzle_size",
+                            "default_value": 0.4,
+                            "type": "float",
+                            "value": "line_width",
```

Comment:
I could make it part of the skin conceptually, but then we'd need to introduce parent settings for all top/bottom and roofing settings.
mesh > skin, infill, walls, etc
skin > top/bottom, roofing